### ISSUE TITLE
Final Fantasy XI - POL viewer jittering in DX12 output modes (v2.75.1/WIP82)

### ISSUE THREAD

**Issue by djc5166** [dege-diosg/dgVoodoo2#93]
v2.75.1 (also up to and including WIP82)

Using DX12 output API (both feature level 11.0 and 12.0) introduces a quick jittering effect on the POL viewer GUI. DX11 output and lower seems to be fine. FFXI itself seems unaffected so far.

I realize this is not a very important issue, I don't think most people even care to use dgvoodoo2 for the POL viewer, but I thought it could be an indicator of a bug that might affect other games.

**Comment by djc5166** [comment]
For the jittering, it doesn't seem to happen in native DX8/DD mode, or any DX11 output modes for me, only DX12.

You can freely resize the POL window, so maybe the render path is based on how large the window is?

If you don't have a retail XI account, you can easily setup a client and connect to a private server. I don't see why any of the renderer stuff would be any different private vs retail at this point.

I don't think I've ever noticed a fog issue, I don't play it too often tbh, but I can check it out.

**Comment by dege-diosg** [comment]
Yes, the window can be resized, but 636x476 was my test case for understanding what happens.
I could reproduce it with D3D11 too for a short while. Some day I'll dig into deeper to see what causes the different rendering paths to be taken.
